libclamav/scanners.c

```diff
@@ -4353,15 +4353,17 @@ cl_error_t cli_magic_scan(cli_ctx *ctx, cli_file_t type)
                 ret = cli_parsegif(ctx);
             break;
 
-        case CL_TYPE_PNG:
+        case CL_TYPE_PNG: {
             if (SCAN_HEURISTICS && (DCONF_OTHER & OTHER_CONF_PNG))
                 ret = cli_parsepng(ctx); /* PNG parser detects a couple CVE's as well as Broken.Media */
             break;
+        }
 
-        case CL_TYPE_JPEG:
+        case CL_TYPE_JPEG: {
             if (SCAN_HEURISTICS && (DCONF_OTHER & OTHER_CONF_JPEG))
                 ret = cli_parsejpeg(ctx); /* JPG parser detects MS04-028 exploits as well as Broken.Media */
             break;
+        }
 
         case CL_TYPE_TIFF:
```

Comment:
If we're bracketing cases here, shouldn't we bracket all of them?

Reply:
This is vestigial from Mickey removing some prototype code that I started her out with.  You're right though in that we don't need the brackets, or we should add them to all -- which would also be kind of nice but a little out of scope for this PR. I'm happy either way.